### PR TITLE
Update remove-a-domain.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note as users need to know what happens to active accounts in o365 when a domain is removed

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3583

File: microsoft-365/admin/get-help-with-domains/remove-a-domain.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ If you have hundreds or thousands of users, use PowerShell to query for all user
 ::: moniker range="o365-worldwide"
 
 > [!NOTE]
-> You can't remove the [".onmicrosoft.com"](https://docs.microsoft.com/microsoft-365/admin/setup/domains-faq) domain from your account.
+> You can't remove the [".onmicrosoft.com"](https://docs.microsoft.com/microsoft-365/admin/setup/domains-faq) domain from your account. When you remove a domain, Office 365 user accounts will revert back to .onmicrosoft.com address as the Primary SMTP/UserprincipalName.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/remove-a-domain.md]:
```suggestion
> You can't remove the [".onmicrosoft.com"](https://docs.microsoft.com/microsoft-365/admin/setup/domains-faq) domain from your account. When you remove a domain, Office 365 user accounts will revert back to the .onmicrosoft.com address as the Primary SMTP/UserprincipalName.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks.

**Pull request merged**